### ISSUE TITLE
Separating training data for validation

### ISSUE THREAD

**Issue by ZhuoyiZhan** [rileymccarty/Average_Joes_Ancient_Artifacts#4]
We know that we have two datasets, archaeologicalSoilData is the test data, lithicExperiementalData is the training data. For machine learning, we need to held back some data from the training date to estimate the performance of the model. And the validation set can be used to see if there is anomalies or if there is significant difference between training and testing data. So for lithicExperiementalData, we want tot take a part of the data as training data and the rest as validation data. 

In terms of separation of data, we want to use 80/20 as the ratio. There should be 80% training data and 20% validation data. The subsets are drawn randomly to keep it fair. After the separation, we can keep the training data and validation in two data frames to facilitate the following steps.

**Comment by rileymccarty** [comment]
I confirmed with Prof. Zhang that the two files are not training and testing, they are positive cases and the negative cases. Change made in the issue to reflect htat
